CODEOWNERS

```diff
@@ -4,7 +4,7 @@
 # Only files explicitly mentioned are guarded by code owners (no * rule).
 
 CODEOWNERS   @gerolf-da @cocreature @stefanobaghino-da
-LATEST       @gerolf-da @bame-da @cocreature @stefanobaghino-da @adriaanm-da
+LATEST       @gerolf-da @bame-da @cocreature @stefanobaghino-da @ray-roestenburg-da @adriaanm-da
```

Comment:
I actually did it initially, but thought it could have made sense for me to keep up with releases. But this is probably the wrong way to do it, I'll remove myself, thanks. 🙇🏻‍♂️

Reply:
Addressed by https://github.com/digital-asset/daml/pull/12847/commits/4d774599497dd315e334abcf89f9975dad67fb26